### PR TITLE
Update copy for incoming calls

### PR DESCRIPTION
**Pull request by CalumMcCall**
#### APK
[Download build #11560](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11560/artifact/build/artifact/wire-dev-PR1753-11560.apk)
[Download build #11561](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11561/artifact/build/artifact/wire-dev-PR1753-11561.apk)
[Download build #11563](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11563/artifact/build/artifact/wire-dev-PR1753-11563.apk)
[Download build #11567](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11567/artifact/build/artifact/wire-dev-PR1753-11567.apk)
[Download build #11576](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11576/artifact/build/artifact/wire-dev-PR1753-11576.apk)
[Download build #11579](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11579/artifact/build/artifact/wire-dev-PR1753-11579.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala [hunk 1]
```diff
@@ -329,13 +329,32 @@ class CallController(implicit inj: Injector, cxt: WireContext, eventContext: Eve
...
-    case (true,  OtherCalling)  => R.string.call_banner_incoming_video
-    case (false, OtherCalling)  => R.string.call_banner_incoming
-    case (_,     SelfJoining)   => R.string.call_banner_joining
-    case (_,     SelfConnected) => R.string.call_banner_tap_to_return_to_call
-    case _                      => R.string.empty_string
+  private lazy val callingUsername: Signal[String] =
+    for {
+      users <- userStorage
+      userId <- currentCall.map(_.caller)
+      data <- users.signal(userId)
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala]: We may have the `caller` signal somewhere in the `CallController`, no?

File: app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala [hunk 2]
```diff
@@ -329,13 +329,32 @@ class CallController(implicit inj: Injector, cxt: WireContext, eventContext: Eve
...
+      name <- teamName
+    } yield {
+       name.getOrElse("") + data.getDisplayName.toUpperCase(getLocale) + " "
+    }
+
+  private lazy val teamName: Signal[Option[String]] =
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala]: converstaionName

Comment (deanrobertcook) [reply app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala]: We may also have this signal somewhere?

File: app/src/main/res/values/glyphs.xml [hunk 3]
```diff
@@ -138,4 +138,5 @@
     <string translatable="false" tools:ignore="UnusedResources" name="glyph__send_timed">\uE735</string>
     <string translatable="false" tools:ignore="UnusedResources" name="glyph__keyboard_timed">\uE736</string>
     <string translatable="false" tools:ignore="UnusedResources" name="glyph__emoji_timed">\uE737</string>
+    <string translatable="false" tools:ignore="UnusedResources" name="glyph__bullet">\u25CF</string>
```
Comment (deanrobertcook) [inline app/src/main/res/values/glyphs.xml]: check out the string `message_footer__status__delivered` and follow a similar pattern

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -360,8 +360,7 @@
 
     <string name="call_banner_joining">CONNECTING…</string>
     <string name="call_banner_outgoing">RINGING…</string>
-    <string name="call_banner_incoming">CALLING…</string>
-    <string name="call_banner_incoming_video">CALLING…</string>
+    <string name="call_banner_incoming">IS CALLING…</string>
```
Comment (deanrobertcook) [inline app/src/main/res/values/strings.xml]: as discussed: best to use separate strings for the two different cases:

```
<string name="abc">%1$s is Calling</string>
<string name="abc">%1$s \u30FB %2$s is Calling</string>
```

**Comment by someoneEsle** [comment]
- [ ] Bug 1
Long group names/contact names are not being truncated correctly. 
647400e0-3ad1-4d9d-9cd7-9a0780029ed6

**Pull request merged**